docs/text.html

```diff
@@ -1,218 +1,331 @@
+            <td>light blue</td>
+            <td>light blue, indigo</td>
+        </tr>
+        <tr>
+            <td style="
+              background-color: #55ff55;
+              color: #000000;
+              width: 10px;
+              height: 10px;
+            "></td>
+            <td>§a</td>
+            <td>light green</td>
+            <td>lime, lime green</td>
+        </tr>
+        <tr>
+            <td style="
+              background-color: #55ffff;
+              color: #000000;
+              width: 10px;
+              height: 10px;
+            "></td>
+            <td>§b</td>
+            <td>light cyan</td>
+            <td>light aqua, turquoise, turquois, light blue</td>
+        </tr>
+        <tr>
+            <td style="
+              background-color: #ff5555;
+              color: #000000;
+              width: 10px;
+              height: 10px;
+            "></td>
+            <td>§c</td>
+            <td>light red</td>
+            <td>pink</td>
+        </tr>
+        <tr>
+            <td style="
+              background-color: #ff55ff;
+              color: #000000;
+              width: 10px;
+              height: 10px;
+            "></td>
+            <td>§d</td>
+            <td>magenta</td>
+            <td>light purple</td>
+        </tr>
+        <tr>
+            <td style="
+              background-color: #ffff55;
+              color: #000000;
+              width: 10px;
+              height: 10px;
+            "></td>
+            <td>§e</td>
+            <td>yellow</td>
+            <td>light yellow</td>
+        </tr>
+        <tr>
+            <td style="
+              background-color: #ffffff;
+              color: #000000;
+              width: 10px;
+              height: 10px;
+            "></td>
+            <td>§f</td>
+            <td>white</td>
+            <td></td>
+        </tr>
+    </table>
+    <p class="subtitle">
+        In <b>Minecraft 1.16</b>, support was added for
+        <b>6-digit</b> hexadecimal colors to specify custom colors other than the 16 default color codes. A new tag can be used to format with these colors. The tag looks like this:
+    </p>
+    <div class="box skript-code-block left-margin">&lt;##hex code&gt;</div>
+    <p class="subtitle">Here's what the tag would look like when used in a script:</p>
+    <div class="box skript-code-block left-margin">
+        send "&lt;##123456&gt;Hey %player%!" to player
+    </div>
+    <p class="subtitle">
+        For information not related to Skript, see
+        <a href="https://minecraft.gamepedia.com/Formatting_codes#Color_codes">Minecraft Wiki page</a
+		>
+		concerning colors. Note that depending on Skript configuration, color
+		codes may do more than just change color of text after them. By default,
+		for backwards compatibility, they clear following styles: magic, bold,
+		strikethrough, underlined, italic. Other styles are not affected, and
+		this feature can be toggled of in config.sk.
```

Comment:
It seems that the indentation from this file has carried over to the website:
![image](https://user-images.githubusercontent.com/29547183/124270410-75742a00-db3c-11eb-8018-b77cb828b567.png)